roster/src/Roster.Core/Domain/EmailAddress.cs

```diff
@@ -14,12 +14,10 @@ public EmailAddress(string email)
         }
 
         // Validate email address format.
-        public static bool Validate(string email)
+        private static void Validate(string email)
         {
             if (!string.IsNullOrEmpty(email))
-                new MailAddress(email);
-                
-            return true;
```

Comment:
No. Since return value is not used, I converted this to void. `MailAddress` constructor would raise an exception in case of error so return value is not needed.